code/Extensions/ProxyDBExtension.php

```diff
@@ -6,12 +6,26 @@
 use SilverStripe\Core\Extension;
 use TractorCow\ClassProxy\Generators\ProxyGenerator;
 use TractorCow\SilverStripeProxyDB\ProxyDBFactory;
+use SilverStripe\Dev\Deprecation;
 
 /**
  * @extends Extension<ProxyDBFactory>
+ *
+ * @deprecated 3.2.0 Will be replaced with an extension on SilverStripe\ORM\Connect\Database
  */
 class ProxyDBExtension extends Extension
 {
+    public function __construct()
+    {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice(
+                '3.2.0',
+                'Will be replaced with an extension on SilverStripe\ORM\Connect\Database.',
```

Comment:
Just small question. 
The comment for `Deprecation::withNoReplacement` says "This is done to surpress deprecation notices by for calls from the vendor dir to deprecated code that projects have no ability to change". But this statement says that there is some ability for replacement. 
Should we rephrase statement?

Reply:
The replacement isn't available until CMS 6, so there's nothing that a user can do in CMS 5 where this deprecation notice shows. The `withNoReplacement()` is used for this reason so as not to annoying the user.